raco/backends/myria/myria.py

```diff
@@ -1914,14 +1902,44 @@ def fire(self, op):
 
 class FillInJoinPullOrder(rules.Rule):
 
-    def __init__(self, pull_order):
+    def __init__(self):
         self._disabled = False
-        self.pull_order = pull_order
+
+    def get_idb_side(self, op):
```

Comment:
This is a misleading name: it doesn't distinguish between "sides" at all, but just tells you if there's an `IDBController` in the current operator's subtree. I think `has_idb_child()` might be a more accurate name.